students/gerry_gabrisch/lesson03/mailroom.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+'''mailroom.py for lesson03, Gerry Gabrisch, started 4/2/2020'''
+
+import sys
+
+#a list of existing donations...
+donations = [('Eric Idle', [200.00, 500.00, 150.50]), 
+             ('John Cleese', [1000, 5000, 20]), 
+             ('Graham Chapman', [10000]), 
+             ('Terry Jones', [50, 600, 1000, 20]), 
+             ('Michael Palin', [100, 200]), 
+             ('Terry Gilliam',[3000])]
+
+def SendThanks(donations):
+    '''adds a donation amount  and sends an email thankyou.  If the donor does not already exist
+       a new element in the donations list will be created'''
+    #get the donor name and convert the first letters of the name to upper case...
+    donor_name = input('Enter a donor name now, or type "List" for a list of donors.>').title()
+
+    #get a new list with all the donors...I did this just to avoid having to add multiple iterators below...
+    donorsOnly = MakeDonorList(donations)     
+    
+    if donor_name == 'List':
+        #if the user types List then call the ListDonors() and print all the names on a new line...
+        ListDonors(donorsOnly)
+    #if this name is not already a donor then make a new donor element and get the donation amount...
+    if donor_name not in donorsOnly and donor_name != 'List':
```

Comment:
use an else here (pair with line 23 if statement), you are checking for "List" comparison again when you could utilize the if / else statement